src/app/order/hooks/useGetExistAddressList.ts

```diff
@@ -0,0 +1,17 @@
+import { getExistAddressList } from '@/apis/order';
+import { ExistAddressInfo } from '@/types/order';
+import { useQuery } from '@tanstack/react-query';
+
+const useGetExistAddressList = () => {
+  const { data, error } = useQuery<ExistAddressInfo[]>(
+    ['existAddressList'],
+    getExistAddressList,
+  );
+
```

Comment:
p2 : 기존배송지에 리스트가 추가되는 경우는 신규입력해서 토글을 선택 후 구매하는 경우 1가지 인 거 같아서, staleTime과 cacheTime을 Infinity로 설정해도 문제 없어 보입니다.
```suggestion

const useGetExistAddressList = () => {
  const { data, error } = useQuery<ExistAddressInfo[]>(
    ['existAddressList'],
    getExistAddressList,
    {
      staleTime: Infinity,
      cacheTime: Infinity,
    },
  );
```